Cargo.toml

```diff
@@ -138,7 +124,7 @@ portpicker = { path = "lib/portpicker" }
 prometheus-parser = { path = "lib/prometheus-parser", optional = true }
 opentelemetry-proto = { path = "lib/opentelemetry-proto", optional = true }
 tracing-limit = { path = "lib/tracing-limit" }
-value = { path = "lib/vrl/value" }
+value = { package = "value", git = "https://github.com/vectordotdev/vrl", rev = "v0.1.0" }
```

Comment:
Just noticing that since we are using the string

> git = "https://github.com/vectordotdev/vrl"

a lot, it would be cool if we could have that as like the equivalent of a const in the toml files for re-use so as to avoid the long string and having to update the revision in all the places when revs change.

Just a thought. Not sure how feasible.

Reply:
Yea, I was hoping there was something like this and didn't see it. It should be easy enough to search/replace since it's a unique string, and in the future we will probably start publishing crates for it.